import-export-cli/integration/api_test.go

```diff
@@ -1270,3 +1273,113 @@ func TestExportImportWebSocketApiFromAsyncApiDef(t *testing.T) {
 		})
 	}
 }
+
+// Import an API and then create a new version of that API by updating the context and version only and import again
+func TestCreateNewVersionOfApiByUpdatingVersion(t *testing.T) {
```

Comment:
Shouldn't this test come to the file devFirst_test.go, since this is related to **apictl init**?

Reply:
Even though we use a dev first approach, the test case itself belongs to Api related test case I thought of adding this here.

Reply:
+1, Let's keep it here since it is easy to track